lib/active_record/connection_adapters/clickhouse_adapter.rb

```diff
@@ -267,7 +267,7 @@ def create_database(name)
         sql = apply_cluster "CREATE DATABASE #{quote_table_name(name)}"
         log_with_debug(sql, adapter_name) do
           res = @connection.post("/?#{@connection_config.except(:database).to_param}", sql)
-          process_response(res)
+          process_response(res, DEFAULT_RESPONSE_FORMAT)
```

Comment:
What do you think about adding tests for `create_database` and `drop_database`?

Reply:
I would love that. Interested in how @PNixx envisions the future of tests here. A CI setup would be nice. For now I tested it only manually on my local setup.